setup.py

```diff
@@ -1,46 +1,65 @@
-  keywords = ['Copernicus', 'CMEMS', 'CLS', 'Motu', 'motu-client-python', 'Dissemination Unit'],
-  
-  author = 'Sylvain MARTY, CLS',
-  author_email = 'smarty@cls.fr',
-  platforms=['any'],
-  
-  url = 'https://github.com/clstoulouse/motu-client-python',
-  license='LGPL',
-  
-  packages=['', 'motuclient'],
-  package_dir = {'': 'src/python', 'motuclient': 'src/python/motuclient'},
-  include_package_data = True,
-  package_data={'motuclient': ['cfg/log.ini',
-                               'cfg/messages.properties']},
-  
-  download_url = 'https://github.com/clstoulouse/motu-client-python/releases/',
-  
-  classifiers = ['Development Status :: 5 - Production/Stable',
-                 'Programming Language :: Python :: 2.7',
-                 'License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)',
-                 'Topic :: Scientific/Engineering :: GIS',
-                 'Environment :: Console',
-                 'Natural Language :: English',
-                 'Operating System :: OS Independent',
-          ]
-)
\ No newline at end of file
+    name='motu-client',
+    version=pom_version.getPOMVersion(),
+    python_requires='>=2.7',
+    description='Extract and download gridded data through a python command line from Motu web server. Used in CMEMS context http://marine.copernicus.eu/',
+    long_description='Motu is a high efficient and robust Web Server which fills the gap between heterogeneous data providers to end users. Motu handles, extracts and transforms oceanographic huge volumes of data without performance collapse. This client enables to extract and download data through a python command line.',
+    keywords=[
+        'Copernicus',
+        'CMEMS',
+        'CLS',
+        'Motu',
+        'motu-client-python',
+        'Dissemination Unit',
+    ],
+
+    author='Sylvain MARTY, CLS',
+    author_email='smarty@cls.fr',
+    platforms=['any'],
+
+    url='https://github.com/clstoulouse/motu-client-python',
+    license='LGPL',
+
+    packages=find_packages,
+    include_package_data=True,
+    package_data={
+        'motuclient': [
+            'cfg/log.ini',
+            'cfg/messages.properties',
+        ],
+    },
+
+    download_url='https://github.com/clstoulouse/motu-client-python/releases/',
+
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Programming Language :: Python :: 2.7',
+        'License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)',
+        'Topic :: Scientific/Engineering :: GIS',
+        'Environment :: Console',
+        'Natural Language :: English',
+        'Operating System :: OS Independent',
+    ],
+
+    entry_points={
+        'console_scripts': [
+            'motu-client = motu-client:main'
```

Comment:
The "dash" character in "motu-client" seems not compatible setuptools / console_scripts (python setup.py sdist).
As this dash character is an issue for a long time, I'll merge your PR and at the same time rename "motu-client" to "motuclient". I will keep the retrocompatibility by keeping script "motu-client" but as soon as "motu-client" will be called a WARNING log message will inform user of this update and insite to use "motuclient".

e.g.:

$ ./motu-client.py --version
2018-10-29'T'15:16:48.702 [WARNING] "motu-client" module is deprecated since version1.8. Now module named "motuclient" without a dash separator between motu and client shall be used instead of "motu-client"., e.g. motuclient.main()
motuclient-python v1.8.0
$ ./motuclient.py --version
motuclient-python v1.8.0

Reply:
I've preserved the dash only for compatibility with your code.
Usually I use `pylint` to validate my code and it throws a lot of error/warning about your sources.

Reply:
Ok perfect ! Yes pylint is a bestpractice, I hope we will use it in the next future ! Thanks again for your updates!